documentation/user-interface/test-editor.md

```diff
@@ -67,6 +67,6 @@ Click the ![Rename button](../../images/working-with-testcafe-studio/settings-ic
 
 ### Delete Actions
 
-Click the ![Remove all actions icon](../../images/user-interface/remove-all-icon.svg) **Remove all actions** button on the editor's toobar to delete all actions from the test.
+Click the ![Remove all actions icon](../../images/user-interface/remove-all-icon.svg) **Remove all actions** button on the editor's toolbar to delete all the actions from the test.
 
-To delete a single action, hover the mouse cursor over the action's row and click the ![Remove icon](../../images/user-interface/remove-big-icon.svg) **Remove action** button to the right of it.
\ No newline at end of file
+To delete a single action, hover the mouse cursor over the action's row and click the ![Remove icon](../../images/user-interface/remove-big-icon.svg) **Remove action** button.
```

Comment:
Don't remove "to the right of it". If we don't write this, a user may understand incorrectly what button to click, because the editor has two Remove buttons with similar icons. One button - on the toolbar, another button appears only when you hover the mouse cursor over the action's row.

Reply:
Ok